feegrant/index.md

```diff
@@ -0,0 +1,239 @@
+---
+parent:
+  title: Understanding the Fee Grant Module
+order: 0
+description: Use the Cosmos SDK feegrant module to grant the use of tokens to pay for fees from one account (the granter) to another account (the grantee).
+---
+
+# Feegrant Module
+
+The [feegrant](https://docs.cosmos.network/v0.44/modules/feegrant/) enables a user / contract / module (the granter) to pay the fees for someone else (the grantee) when they want to broadcast a transaction on the blockchain. A key feature of feegrant is that the granter retains full access to their tokens, and is able to revoke the allowance at any time.
+
+## Use Feegrant to Grant an Allowance
+
+An often discussed use case for the feegrant module is that blockchains or smartcontracts can allow new users to start interacting with the chain or contract without having to acquire tokens first, greatly improving the onboarding experience.
+
+There are two [fee allowance types](https://docs.cosmos.network/v0.44/modules/feegrant/01_concepts.html#fee-allowance-types) that the freegrant module implements: `BasicAllowance` and `PeriodicAllowance`, the latter allows you to setup a periodic allowance to another user. In this tutorial, you will set up two tokens in your blockchain: one default token called `stake`, which will be used for fees, and another called `kudos`, for sending to your friends.
+
+You will learn how to spin up a single node network using the simulation application in Cosmos SDK (`simapp`), let Charlie grant a `BasicAllowance` to Bob, and then have Bob send `kudos` tokens to Charlie, while having zero `stake` to pay for fees.
```

Comment:
Not sure if you want to have this in the repo: when _creating_ users, A, B, C and so on we try to keep the names feminine / masculine / fem / masc and so on. That would mean Alice, Bob, Carol (or other feminine name), Dave (or other masculine name)...

Reply:
Additionally, to be predictably conventional, I would introduce the names in order:

> ...single node network ... validated by **Alice**, let **Bob** grant an allowance to **Carol** and then have Carol send...

Like this, no reader will think "Did I miss a mention of Alice earlier?"

Reply:
Makes total sense, thanks! I'll change it in the next commit.